src/components/LoginForm.js

```diff
@@ -76,7 +77,13 @@ export default class LoginForm extends React.Component {
         if (err) {
           this.setState({ errorMessage: err.message });
         } else {
-          this.history.pushState(null, this.props.redirectTo || '/');
+          var router = Context.getInstance().getRouter();
+
```

Comment:
Why this new line here? The rest of the declarations doesn't seem to be *grouped* more than that two of them are routes?